modules/jpa/pom.xml

```diff
@@ -19,6 +19,18 @@
       <groupId>io.oasp.java.modules</groupId>
       <artifactId>oasp4j-jpa-dao</artifactId>
     </dependency>
+    <dependency>
+     <groupId>io.oasp.java.modules</groupId>
+  <artifactId>oasp4j-jpa-envers</artifactId>
+      </dependency>
+   <dependency>
+      <groupId>org.hibernate</groupId>
+      <artifactId>hibernate-envers</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+    <artifactId>spring-context</artifactId>
```

Comment:
nice to have would be a formatting ([Ctrl][Shift][f]) as the indentation is a little mixed.